.github/workflows/ci.yml

```diff
@@ -198,6 +198,18 @@ jobs:
       - name: "Check we're not using a -dev version of the embedded compiler"
         run: jq -r '.["compiler-version"]' package.json | grep -qv -- '-dev$'
 
+      - name: Publish optional dependencies
+        env:
+          NODE_AUTH_TOKEN: '${{ secrets.NPM_TOKEN }}'
+        run: |
+          cd npm/linux-arm64 && npm_config_platform=linux npm_config_arch=arm64 npx ts-node ../../tool/prepare-optional-release.ts && npm publish && cd ../..
```

Comment:
Not sure if this is ok, since it manually sets the `npm_config_*` environment variables.

Reply:
Can you pass these in as CLI parameters to `prepare-optional-release.ts` instead?

Reply:
Just changed it, please have a review.

Reply:
Tried CLI parameters, check https://github.com/sass/embedded-host-node/pull/158/commits/e6c9ea2557a7e341eaa5559689dcea2ca7e25211, but I think add a `prepare-optional` npm scripts is more simple and clean: `npm run prepare-optional --platform=linux --arch=arm64 && npm publish ./npm/linux-arm64`.